diff.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE IF EXISTS public.connectors
```

Comment:
This should be renamed, and under `supabase/pending/`, if it's an unapplied migration.